src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/messaging/PrisonerReceivedIntoPrisonEventService.kt

```diff
@@ -40,16 +40,19 @@ class PrisonerReceivedIntoPrisonEventService(
       prisonAdmittedTo = prisonId,
       eventDate = eventOccurredAt,
     )
+
+    /*
```

Comment:
These are the TODO's that I had previously added to `PefCiagKpiService` (which are still there at the moment)
The TODOs are based on the originally added ones, plus [the slack conversation with Aliki](https://mojdt.slack.com/archives/C05E3G0PU65/p1736266592070669) about it yesterday